pom.xml

```diff
@@ -88,6 +88,8 @@
     <mockito.version>4.11.0</mockito.version>
     <mockito-scalatest.version>1.17.14</mockito-scalatest.version>
     <netty.version>4.1.107.Final</netty.version>
+    <netty-tcnative.version>2.0.65.Final</netty-tcnative.version>
```

Comment:
where is the consumer of this property?

Reply:
This should have been removed - will come in a follow up work to add openssl support